MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```
@@ -33,6 +36,10 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		}
 	}
 	
+	func show(_ viewModel: FeedErrorViewModel) {
+		errorView.isHidden = false
```

The implementation should check if the viewModel contains an error message.

If it does, show the error.
Else, hide the error.

And it should use the provided methods `errorView.show(message: message)`  and `errorView.hideMessage()`.